scripts/travis/install-tensorflow.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+set -e
+
+LOCAL_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+if [ ! -z "$DEB_BUILD" ]; then
+    echo "Skipping for deb build"
```

Comment:
Not necessary since https://github.com/NVIDIA/DIGITS/commit/543ebb6340b1b656075fcd3a835af4b315961e3b